### ISSUE TITLE
It should be harder to discard a partially-written toot

### ISSUE THREAD

**Issue by SamirTalwar** [feditext/feditext#132]
**Is your feature request related to a problem? Please describe.**

It's too easy to cancel a toot.

**Describe the solution you'd like**

I would like to be prompted if I really meant to. Ideally, the prompt wouldn't show if I haven't typed anything (including when replying).

**Describe alternatives you've considered**

Make it easy to undo cancellation for a minute or so.

**Additional context**

I don't think I'm the only clumsy one out there.

**Comment by VyrCossont** [comment]
Related to #133 in that users will want to be able to save cancelled posts as drafts on command.

**Comment by SamirTalwar** [comment]
Agreed, but focused on accidentally cancelling, rather than purposefully deciding not to send something immediately.
